jcl/src/jdk.management/share/classes/com/ibm/lang/management/internal/CpuUtilizationHelper.java

```diff
@@ -64,7 +64,12 @@ synchronized double getSystemCpuLoad() {
 
 		/* calculate using the most recent value in the history */
 		if ((latestTime.getTimestamp() - interimTime.getTimestamp()) >= MINIMUM_INTERVAL) {
-			cpuLoad = calculateCpuLoad(latestTime, interimTime);
+	            if (latestTime.getCpuLoad() == -1) {
+		        cpuLoad = calculateCpuLoad(latestTime, interimTime);
+                    } else {
+		        cpuLoad = (double)latestTime.getCpuLoad()/100;
```

Comment:
Spaces needed around mathematical operators for readability. Similarly elsewhere in this PR.

Reply:
Change applied in ffdf65f8be493319c906b24c60b752782364cb35